src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,102 @@
 package core.basesyntax;
 
+import java.util.LinkedList;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private int initialCapacity = 16;
+    private float loadFactor = 0.75f;
+    private int size = 0;
```

Comment:
Using public access modifiers everywhere is not recommended. If the field `size` is only used inside this class, consider making it private to follow the encapsulation principle.